components/log/log_freertos.c

```diff
@@ -52,7 +52,11 @@ void esp_log_impl_unlock(void)
 
 char *esp_log_system_timestamp(void)
```

Comment:
The description of this function in esp_log.h needs to be updated to match

Reply:
Would you prefer changing the name of the function or splitting it into esp_log_system_timestamp and esp_log_system_datetimestamp? The xTaskGetSchedulerState portion would be put into a static function for use between both.

Reply:
@Molorius Good point! That is probably a cleaner solution, thank you for pointing it out.

However if it becomes less clean to implement in some way, I'm OK with the current implementation - it just needs to be noted in the header that it might return a date+time depending on config.

Reply:
I separated the datetime elements into a separate function and updated esp_log.h to match.